libsolidity/formal/SymbolicState.h

```diff
@@ -189,13 +189,13 @@ class SymbolicState
 	BlockchainVariable m_tx{
 		"tx",
 		{
-			{"blockhash", std::make_shared<smtutil::ArraySort>(smtutil::SortProvider::uintSort, smtutil::SortProvider::uintSort)},
 			{"block.chainid", smtutil::SortProvider::uintSort},
 			{"block.coinbase", smt::smtSort(*TypeProvider::address())},
```

Comment:
Again in this mapping, coinbase/origin is address, and sender is uint.

Reply:
will change